rest/model/codegen.go

```
@@ -372,8 +382,8 @@ func validateFieldTypes(fieldName, inputType, outputType string) error {
 	outputType = stripPackage(outputType)
 	if !strings.Contains(outputType, inputType) && !strings.Contains(inputType, outputType) {
 		// this should ideally be a more sophisticated check to ensure that complex types are convertible
-		// to each other, but for now we rely on the naming convention to find obvious type errors
-		return errors.Errorf("DB model field '%s' has type '%s' which is incompatible with REST model type '%s'", fieldName, inputType, outputType)
+		// to each other, but for now we rely on the naming convention to find potential type errors
+		return errors.Errorf("DB model field '%s' has type '%s' which may be incompatible with REST model type '%s'", fieldName, inputType, outputType)
```

Why do we choose to return an error here instead of logging like we do above?